docs/docs/references/configuration/cli/trivy_kubernetes.md

```diff
@@ -36,11 +37,12 @@ trivy kubernetes [flags] { cluster | all | specific resources like kubectl. eg:
       --components strings                specify which components to scan (workload,infra) (default [workload,infra])
       --config-data strings               specify paths from which data for the Rego policies will be recursively loaded
       --config-policy strings             specify the paths to the Rego policy files or to the directories containing them, applying config files
-      --context string                    specify a context to scan
```

Comment:
I'm sorry I was on holiday so I haven't followed the discussion, but what is the motivation of removing the `--context` flag and making it a CLI argument?

Reply:
@knqyf263 we decided to do this in order to keep the convention with other trivy command which is:

trivy <target> <target name>

examples:
trivy image <image name>
trivy k8s <context>

Reply:
if context not specified we will use the default